src/lib/index.jsx

```diff
@@ -213,6 +214,8 @@ class OtpInput extends Component<Props, State> {
     } else if (e.keyCode === RIGHT_ARROW || e.key === 'ArrowRight') {
       e.preventDefault();
       this.focusNextInput();
+    } else if (e.keyCode === SPACEBAR || e.key === 'Space' || e.key === ' ' || e.key === 'Spacebar') {
```

Comment:
Hi @utkarsh22garg, one last thing.`e.key === 'Space'` can be removed. The rest looks good!

Reply:
Hi @apollonian. I have made the appropriate changes.

Reply:
Thank you so much!